### ISSUE TITLE
new line in ol.style.Text is ignored while zooming

### ISSUE THREAD

**Issue by 10lojzo** [Viglino/OL3-AnimatedCluster#20]
When defining the text style with new line "\n" in text attribute, the text appears inline while zooming, both in cluster and original feature.

`text: new ol.style.Text(
{	text: size.toString() + "\nnew line",
	fill: new ol.style.Fill(
	{	color: '#fff'
	})
})`

I tweaked this example changing just the code above.
http://viglino.github.io/OL3-AnimatedCluster/

**Comment by Viglino** [comment]
Yes it seems there is an issue when drawing the text with the Openlayers function.
I'll try to see if there is a fix to do that.

**Comment by novumcoder** [comment]
This one is still open. I still have this issue with OL 5.3. Any news?

**Comment by Viglino** [comment]
Hello,
The lib use the Openlayers drawFeature ([ol/render/VerctorContext.drawFeature(feature, style)](https://openlayers.org/en/latest/apidoc/module-ol_render_canvas_Immediate-CanvasImmediateRenderer.html)). It seems this function doesn't care the new line when drawing...
NB: this repository is now obsolete. It is now part of the [ol-ext project](https://github.com/Viglino/ol-ext), create a new issue in [the ol-ext issue tracker](https://github.com/Viglino/ol-ext/issues).
@+
